### PR TITLE
Fix kebab menu locator for add capacity UI test

### PR DESCRIPTION
**Pull request by am-agrawa**
Failure seen in https://url.corp.redhat.com/9c00079
When existing locator was checked, the data test attribute has changes in and it also has a missing closing bracket.

Before-
`"kebab_storage_cluster": ('button[data-test-id="kebab-button"', By.CSS_SELECTOR),`
After-
`"kebab_storage_cluster": ('button[data-test="kebab-button"]', By.CSS_SELECTOR),`

### CODE REVIEW

**Review by prsurve — DISMISSED** [review]

**Review by DanielOsypenko — APPROVED** [review]

File: ocs_ci/ocs/ui/add_replace_device_ui.py [hunk 1]
```diff
@@ -30,9 +30,12 @@ def add_capacity_ui(self):
             self.do_click(self.add_capacity_ui_loc["ocs_operator"])
             self.do_click(self.add_capacity_ui_loc["storage_cluster_tab"])
         time.sleep(1)
+        logger.info("Click on kebab menu of Storage Systems")
         self.do_click(
             self.add_capacity_ui_loc["kebab_storage_cluster"], avoid_stale=True
         )
+        self.take_screenshot()
```
Comment (DanielOsypenko) [inline ocs_ci/ocs/ui/add_replace_device_ui.py:37]: we may provide description as an arg to self.take_screenshot. This way it is easier to find correct screenshot, not guessing and calculating the capture time.

**Review by DanielOsypenko — DISMISSED** [review]

**Comment by am-agrawa** [comment]
Test passed here- https://url.corp.redhat.com/53f2123

File: ocs_ci/ocs/ui/add_replace_device_ui.py (hunk 1 above)
Comment (am-agrawa) [reply ocs_ci/ocs/ui/add_replace_device_ui.py:37]: sure, it can be done later if necessary

**Review by prsurve — APPROVED** [review]

**Review by ebondare — APPROVED** [review]

**Review by ebenahar — APPROVED** [review]

**Comment by am-agrawa** [comment]
Backport is not needed, test passed 88% with ODF 4.15

**Pull request merged**